### DISCUSSION TITLE
Add to list dialog order changes with use

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#9094] [Other Feature Feedback, Questions, & Ideas] [answered]
I migrated 136 ⭐️ into 3 lists.

126 items went into one list.

Generally, the most recently modified list would be the top item in the next use of the dialog.

If I added an item to that main list and another list, it didn't matter in which order I checked the two lists, the lesser used list would be moved to the top for the next appearance of the dialog.

The effective randomness of this list meant that I couldn't "fast add" because muscle memory was broken each time the next dialog appeared. It wasn't the first item (by weight), or the nth item (by alphabet), but based on whether or not the previous item was also added to some other list.

I understand the idea of favoring the most recently selected list, but if that's the goal, then the check order in the most recently used dialog should matter and enable me to guide the next dialog.

But, personally, given the choice of:
* most recently checked item in previous dialog (which might not be 5s ago, but could be in a different window, on a different computer, on a different day/week/month)
* alphabetical
* item count

I'd prefer either of the last two over the first one.

Of course, if I could actually provide a fixed order that isn't alphabetical, that would also be fine. But, typically if I'm given alphabetical order and need to force an order, i'll just add numbers at the beginning to force the order, so alphabetical is good enough for that use case.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note and suggestions @jsoref — we've had a bunch of requests around sorting/ordering for lists, and I'll add your +1 to our tracking issues to make this better!
